source/extensions/filters/http/lua/lua_filter.h

```diff
@@ -308,6 +308,8 @@ class StreamHandleWrapper : public Filters::Common::Lua::BaseLuaObject<StreamHan
   State state_{State::Running};
   std::function<void()> yield_callback_;
   Http::AsyncClient::Request* http_request_{};
+
+  absl::flat_hash_map<std::string, Envoy::Common::Crypto::CryptoObjectPtr> public_key_storage_;
```

Comment:
I think it's fine but can you add a comment that this map is never removed from?